server/src/main/java/org/apache/iotdb/db/service/thrift/impl/DataNodeRegionManager.java

```diff
@@ -142,7 +143,7 @@ private TSendPlanNodeResp executeDataInsert(DataRegionId dataRegionId, InsertNod
       }
 
       ConsensusWriteResponse writeResponse =
-          executePlanNodeInConsensusLayer(dataRegionId, insertNode);
+          FragmentInstanceDispatcherImpl.writePlanNode(dataRegionId, insertNode);
```

Comment:
Better to put that static method in this class, it makes sense that DataRegionManager hold the method about writing.

Reply:
Fixed.